src/Infrastructure/LeanCode.PeriodicService/IPeriodicAction.cs

```diff
@@ -8,5 +8,5 @@ public interface IPeriodicAction
 {
     CronExpression When { get; }
     bool SkipFirstExecution { get; }
-    Task ExecuteAsync(CancellationToken stoppingToken);
+    Task ExecuteAsync(CancellationToken cancellationToken);
```

Comment:
stoppingToken here was correct

Reply:
Hm here I'm not so sure, do we stick with `BackgroundService`s `stoppingToken` or with ours `cancellationToken`? I'm leaning towards renaming this to `cancellationToken` as this will be used in our apps and will be more consistent with our naming.

Reply:
This `CancellationToken` is much different than "normal" `CancellationToken`. It does not represent cancelling the operation - it notifies you about stopping the app. We should keep the original name.

Also, this is not the only place where `CancellationToken` does not represent `cancellation`.